.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    name: Build rocks
+    runs-on: ubuntu-22.04
+    outputs:
+      rockcraft-out: ${{ steps.rockcraft.outputs.rock }}
+    strategy:
+      matrix:
+        rock:
+          - api-server
+          - persistenceagent
+          - scheduledworkflow
+          - viewer-crd-controller
+          - visualisation-server
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+
+      - name: Setup ROCKs environment
+        uses: canonical/craft-actions/rockcraft-pack@main
+        id: rockcraft
+        with:
+          path: ./${{ matrix.rock }}
+
+      - name: Upload rock as artefact
+        uses: actions/upload-artifact@v3
+        with:
+            name: kfp-operators-rocks
+            path: ${{ steps.rockcraft.outputs.rock }}
```

Comment:
The only concern here is the size of the ROCK. Not sure if it makes sense to upload ROCKs as artefacts.

Reply:
Well, I built the rocks in this repository and the artefact is ~1.5GB, so I guess not a big problem. I'm not sure about the artefact size limit, though.